### PR TITLE
Chore(update): Sync experiment and rbac permissions

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: udit <udit.gaurav@mayadata.io>

#### What this PR does / why we need it:

- This PR is for syncing the experiment and RBAC permission from different experiment templates in this repository.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] DCO signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

**Comment by uditgaurav** [comment]
### Running a local helm lint

```bash
udit@ubuntu ~/g/s/g/l/litmus-helm> helm lint ./charts/kubernetes-chaos
==> Linting ./charts/kubernetes-chaos

1 chart(s) linted, 0 chart(s) failed
udit@ubuntu ~/g/s/g/l/litmus-helm> helm lint ./charts/litmus-portal/
==> Linting ./charts/litmus-portal/

1 chart(s) linted, 0 chart(s) failed
udit@ubuntu ~/g/s/g/l/litmus-helm> helm lint ./charts/litmus/
==> Linting ./charts/litmus/

1 chart(s) linted, 0 chart(s) failed
```

**Comment by ksatchit** [comment]
The stable helm charts is deprecated. We should be moving to helm 3 as part of the build process

**Review by ksatchit — APPROVED** [review]

File: charts/kubernetes-chaos/templates/pod-delete.yaml [hunk 1]
```diff
@@ -18,13 +18,23 @@ spec:
...
+          - "argoproj.io"
           - "batch"
           - "litmuschaos.io"
         resources:
           - "deployments"
           - "jobs"
           - "pods"
           - "pods/log"
+          - "replicationcontrollers"
+          - "deployments"
```
Comment (hochuenw-dd) [inline charts/kubernetes-chaos/templates/pod-delete.yaml:31]: this gets confusing becuase you've already listed deployments in https://github.com/litmuschaos/litmus-helm/pull/66/files#diff-854116dbd4691d0a3839388884f44969b4ff41e5e2d62c3579a93868380721b2R26

This is becuase the way it defines the policy is wrong. It should be something like

```
- apiGroups: [""]
  resources: ["pods","pods/log","events"]
  verbs: ["create","list","get","patch","update","delete","deletecollection"]
- apiGroups: ["batch"]
  resources: ["jobs"]
  verbs: ["create","list","get","patch","update","delete","deletecollection"]
- apiGroups: ["apps"]
  resources: ["deployments"]
  verbs: ["create","list","get","patch","update","delete","deletecollection"]
- apiGroups: ["apps"]
  resources: ["replicasets"]
  verbs: ["list","get"]
- apiGroups: ["litmuschaos.io"]
  resources: ["chaosengines","chaosexperiments","chaosresults"]
  verbs: ["create","list","get","patch","update","delete","deletecollection"]
- apiGroups: ["argoproj.io"]
  resources: ["rollouts"]
  verbs: ["list","get"]
```

**Pull request merged**